src/backend/base/langflow/components/vectorsearch/__init__.py

```diff
@@ -9,10 +9,12 @@
 from .VectaraSearch import VectaraSearchComponent
 from .WeaviateSearch import WeaviateSearchVectorStore
 from .pgvectorSearch import PGVectorSearchComponent
+from .Couchbase import CouchbaseSearchComponent # type: ignore
```

Comment:
Is it possible to handle this better? Ideally, you should not have to ignore this as the other imports don't

Reply:
I get this error
src/backend/base/langflow/components/vectorsearch/__init__.py:12: error: Skipping analyzing "langflow.components.vectorsearch.Couchbase": module is installed, but missing library stubs or py.typed marker  [import-untyped]

If i do not ignore the type

Reply:
Even other vectore stores have added this ignore in their respective files